test/bin/sandman/test_sweep.py

```diff
@@ -46,6 +47,8 @@
 
 config, idm = generate_config(Executable.SANDMAN)
 
+_HGI_FARM_SANDMAN_CONFIG_LOCATION = "/software/hgi/installs/vault/etc"
```

Comment:
Is it not possible to have this at a higher level so it is one place and shared by test_persistence.py?